src/interface/manager.ts

```diff
@@ -7,9 +7,31 @@ import { Meta } from './meta';
 export abstract class Manager {
 
     protected name: string;
+    private cached: { [key: string]: any };
 
     constructor(name: string) {
         this.name = name;
```

Comment:
The name seems to e useless in this context.

Reply:
We need name property in `manager` classes for deps generation